README.md

```diff
@@ -10,6 +10,110 @@ Centos 6
 
 Centos 7?
 
+## Usage
+
+### Resource
+
+```
+    python_webapp "python_app_name" do
+      option "option_value"
+    end
+```
+
+Option                  | Type      | Description                                   | Default Value
+----------------------- | --------- | --------------------------------------------- | -------------
+`:create_user`          | Boolean   | Creates a user on the sever                   | `false`
+`:path`                 | String/Nil| Path to application                           | `nil` with a calculated default.
+`:owner`                | String    | Owner of application files (user)             | `chef`
+`:group`                | String    | Owner of application files (group)            | `chef`
+`:repository`           | String    | URL to git repository                         | `cannot be nil`
+`:revision`             | String    | Branch or Commit Hash of Repo                 | `master`
+`:virtualenv_path`      | String    | Path to python virtualenv                     | `nil` with a calculated default.
+`:config_template`      | String/Nil| Path to app config template                   | `nil` with a calculated default.
+`:config_destination`   | String/Nil| Path to app config location on server         | `nil` with a calculated default.
+`:config_vars`          | Hash      | Variables for the config template             | `Cannot be nil if config_destination is set, can be empty`
+`:requirements_file`    | String/Nil| Application's dependencies file.              | `nil` with a calculated default.
+`:django_migrate`       | Boolean   | Whether or not to run django migrations       | `false`
+`:django_collectstatic` | Boolean   | Whether or not to run django collectstatic    | `false`
+`:interpreter`          | String    | Python command (python, python3, etc)         | `python`
+`:gunicorn_port`        | Int/Nil   | Port to run gunicorn on                       | `nil`
+
+For more information on some of these resources (and the sane defaults
+python-webapp sets in case of certain values being) see the 'Notes' section
+below.
+
+### Example
+
+**SETUP:** Before you can use the python-webapp cookbook add the following
+informtion to the corresponding files:
+
+* Add `depends 'python-webapp'` to your metadata.rb.
+* Add `cookbook 'python-webapp', git:
```

Comment:
Eventually we'll publish this to the supermarket, which will make this line irrelevant.